kuma/javascript/src/header/header.jsx

```diff
@@ -1,5 +1,6 @@
 //@flow
 import * as React from 'react';
+import { useMemo } from 'react';
```

Comment:
Why not use `React.useMemo` on line 101? Is this a necessary hack because of Flow?

Reply:
It is convention that hooks are used "bare" like this. There may also be tools that depend on it